gnoi/os/manager_test.go

```diff
@@ -54,7 +54,7 @@ func TestSetRunning(t *testing.T) {
 			manager := NewManager("new")
 			err := manager.SetRunning(test.version)
 			if (err == nil) == test.wantsErr {
-				t.Errorf("Error running OS: Error is %w", err)
+				t.Errorf("Error running OS: Error is %v", err)
```

Comment:
What version of go did you run into issues with `%w`?

Reply:
It was introduced in 1.13

Reply:
https://blog.golang.org/go1.13-errors

Reply:
my bad, I used my personal laptop and did not check the go version, nice you spotted this :)